values.yaml

```diff
@@ -336,4 +336,24 @@ container_images:
         repository: oauth2-proxy/oauth2-proxy
         tag: v7.9.0@sha256:37c1570c0427e02fc7c947ef2c04e8995b8347b7abc9fcf1dbb4e376a4b221a7
   
-enable_chisel_proxy_protocol: false
\ No newline at end of file
+enable_chisel_proxy_protocol: false
+
+nodePorts:
```

Comment:
could we change this to be snake case? `node_ports` I believe snake case is the convention we are using in most places.